Hymns/Tags/TagSheetView.swift

```diff
@@ -88,7 +90,7 @@ struct TagSheetView: View {
                         }.padding(.top)
                     }
                     Spacer()
-                }.offset(y: self.kGuardian.slide).animation(.easeInOut(duration: 1.0))
+                }.offset(y: self.kGuardian.slide)//.animation(.easeInOut(duration: 1.0))
```

Comment:
This animation was causing all kinds of problems. It was applying to everything on the view. So for instance every time you pull up tags. The Empty Tag Illustration photo was easing out even though it should never appear in the first place. Did you mean to apply this animation to something in particular?